kit/src/components/message/mod.rs

```diff
@@ -693,7 +693,16 @@ pub fn IdentityMessage(cx: Scope<IdentityMessageProps>) -> Element {
                     onpress: move |_| {
                         ch.send(IdentityCmd::SentFriendRequest(identity.did_key().to_string(), state.read().outgoing_fr_identities()));
                     },
-                    text: get_local_text_with_args("friends.add-name", vec![("name", identity.username())]),
+                    icon: if disabled {
+                        Icon::Check 
```

Comment:
```suggestion
                        Icon::Check
```

Reply:
linter was complaining about it

![image](https://github.com/Satellite-im/Uplink/assets/29093946/6f78d94c-e60b-4a22-a5b0-7e1774e9de13)